scripts/installscripts/buster-install-default.sh

```diff
@@ -861,7 +861,12 @@ sudo amixer cset numid=1 100%
 # delete the global.conf file, in case somebody manually copied stuff back and forth
 # this will be created the first time the Phoniebox is put to use
 # by web app or RFID
-rm /home/pi/RPi-Jukebox-RFID/settings/global.conf
+GLOBAL_CONF=/home/pi/RFID/settings/global.conf
```

Comment:
Sorry, didn't catch that when I first looked at it, but the path is not correct, it should be `/home/pi/RPi-Jukebox-RFID/settings/global.conf`

Reply:
Oh yeah, you are right. I will fix that. Stupid copy-paste failure.

Reply:
Updated with 4116795